### ISSUE TITLE
In Dappled Groves - Leaf decay [modded Trees and Vanilla]

### ISSUE THREAD

**Issue by Burgersim** [Aetherial-Labs/Issue-Reporting#6]
Hi there, I thought maybe I'd open an issue just for reference. It seems the leaf decay isn't working right anymore since I installed the mod. As a side note, also tree felling for thinner trees (like alder trees form wildcraft trees) doesn't seem to work at all, so I can only cut them down one block at a time for now.

I tried it on Linux and Windows btw so the problem doesn't seem to be an OS-specific issue.

**Comment by VinterNacht** [comment]
The issue with the leaves isn't working correctly in vanilla either. I've been playing a vanilla play-thru to refamiliarize myself, and I've noticed this consistently. I've also noticed it from other individuals who aren't using IDG, so I don't believe it's related. I believe it relates to their update on how tree felling works.

I will look into the thin trees not being felled. That's not something that should be happening.
